demos/paint_your_dreams_demo/README.md

```diff
@@ -15,10 +15,19 @@ Check "Generate endlessly" to generate a new image just after the previous has a
 
 If you want a **quick setup** without manually installing dependencies, use the provided installer scripts. These scripts will **automatically configure** everything needed to run the Paint your Dreams Demo.
 
+Before running these scripts, make sure you’re inside the `setup/` folder:
```

Comment:
Hmm, that may not be the case. I used it a few times and never from the setup folder. If you download it directly it's in your Downloads ;)